x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/request/cohere/CohereEmbeddingsRequestEntity.java

```diff
@@ -29,14 +31,6 @@ public record CohereEmbeddingsRequestEntity(
 
     private static final String SEARCH_DOCUMENT = "search_document";
     private static final String SEARCH_QUERY = "search_query";
-    /**
```

Comment:
I'm going to go with a switch case here instead because that seems a little cleaner now that we have a value we don't want to allow.